README.md

```diff
@@ -29,9 +30,21 @@ In your `apollo.config.js` file:
 
 ```js
 const { config } = require("vscode-apollo-relay").generateConfig()
+// If you want to debug the config, uncomment the following line
+// console.log(JSON.stringify(config))
+// You have to restart Visual Code after installing and check the output of Apollo GraphQL (next to the Terimal window)
 module.exports = config
 ```
 
+If you  use [relay-config], don't forget to specify the language 
+```javascript
+// relay.config.js
+module.exports = {
+  ..............
+  language: 'typescript',
+}
+```
```

Comment:
This section may not be necessary, as I would normally expect this to be implied and applies to all of the config entries. Perhaps the docs should be amended to instead state something like “be sure to specify _all_ configuration required to run `relay-compiler` in your `relay.config.js` file”?